precode.sh

```diff
@@ -7,37 +7,77 @@
 #   dir3
 #       dir4
 
+mkdir /tmp/task
```

Comment:
Создаем относительно текущей директории, а не `/tmp`
Можно одной командой
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```